### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by scheuerkev**
Hi ! 
Here you can find a first bunch in french translations. As you can see, I've performed i18n settings in relevant JSON global files. 
:)
Regards

### CODE REVIEW

File: fr/arrays/for-each.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+```typescript
+numbers.forEach((number) => console.log(number * 2));
+```
+
+La méthode `forEach` ne modifie pas le tableau original. Elle itère simplement les éléments les uns après les autres et exécute le colback sur chacun d'entre eux.
+
+{% hint style="warning" %}
+La méthode `forEach()` n'est pas exécutée dans le cas d'une instruction vide.
+{% endenta %}
```
Comment (sumn2u) [inline fr/arrays/for-each.md]: Oops, this one is an internal tag and shouldn't be translated. Can you change it back to  `{% endhint %}`?

Comment (scheuerkev) [reply fr/arrays/for-each.md]: My pleasure ! Sorry for this ! Changed back and commited 🚀

**Review by sumn2u — APPROVED** [review]
LGTM

**Comment by sumn2u** [comment]
This PR fixes #56 issue.

**Pull request merged**